pwndbg/dbg/gdb.py

```diff
@@ -22,6 +58,8 @@ def setup(self):
         load_gdblib()
         load_commands()
 
+        from pwndbg.gdblib import prompt
```

Comment:
Why did you move the import in here?

Reply:
Importing `pwndbg.gdblib.prompt` triggers a chain of imports that ends with importing `pwndbg.commands.comments`, which uses `pwndbg.command.ArgparsedCommand`.

Before this PR, this wouldn't really be a problem, but this PR changes the way commands are added, such that it depends on the `pwndbg.dbg` object to be properly set up, as adding new commands relies on calling `pwndbg.dbg.add_command()`.

Because importing `pwndbg.dbg.gdb` is part of the process of setting up `pwndbg.dbg` under GDB, we can't really import anything that depends on it already being set up at the root of the module. It's fine importing it in `setup()`, though, as it only gets called after the object has been set up. In fact, the `setup()` exists precisely because we want to do most of our setting up after that object has been set up.

Reply:
Ah, right. Another way of thinking about it is we shouldn't do any gdblib imports until after `load_gdblib` is called. Can you add a comment about this here? It's something non-obvious we should call out.